Makefile

```diff
@@ -484,10 +484,19 @@ test-and-cover: $(RESOURCES_DOT_GO)
 		source envvars.bash; \
 	fi && \
 	for pkg in $$(cat testpackages.txt); do \
-		go test -v -tags="headless" -covermode=count -coverprofile=profile_tmp.cov $$pkg || exit 1; \
+		go test -race -v -tags="headless" -covermode=count -coverprofile=profile_tmp.cov $$pkg || exit 1; \
```

Comment:
Added race detection to test-and-cover so that we find out about new issues sooner rather than later.